sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -0,0 +1,166 @@
+# Azure SDK Dynamic JSON samples
+
+Some Azure SDK APIs return values that hold raw JSON, for example [protocol methods](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md).  JSON content can be accessed using any JSON parser, such as `JsonDocument`, but use of these APIs can result in code that is difficult to read and may obscure the author's intent.  To improve the developer experience, Azure.Core provides a [dynamic](https://learn.microsoft.com/dotnet/csharp/advanced-topics/interop/using-type-dynamic) layer over JSON APIs.
+
+## Accessing Response Content
+
+### Get `dynamic` from `Response`
```

Comment:
I recommend against markup in headers. Generated stubs tend to be inconsistent depending on what's processing it.

Reply:
Cool, thank you, will fix!